content/millennium/r4/clinical/diagnostics/diagnostic-report.md

```diff
@@ -139,6 +137,11 @@ The common [errors] and [OperationOutcomes] may be returned.
 
 ## Retrieve by id
 
+Retrieve an individual DiagnosticReport by its id:
```

Comment:
since this is referring to the concept, let's capitalize "ID"

if it were specifying the attribute as `Patient.id`, I'd be good with leaving it lowercase

Reply:
re-formatting nowish